Frameworks/Foundation/NSURLProtocol_file.mm

```diff
@@ -46,28 +52,19 @@ + (BOOL)canInitWithRequest:(id)request {
 }
 
 - (id)initWithRequest:(id)request cachedResponse:(id)response client:(id)client {
-    [super initWithRequest:request cachedResponse:response client:client];
-
-    _modes = [[NSMutableArray arrayWithObject:@"kCFRunLoopDefaultMode"] retain];
+    if (self = [super initWithRequest:request cachedResponse:response client:client]) {
+        id url = [_request URL];
+        TraceVerbose(TAG, L"Loading %hs", [[url absoluteString] UTF8String]);
 
-    id url = [_request URL];
-    TraceVerbose(TAG, L"Loading %hs", [[url absoluteString] UTF8String]);
-
-    _path = [[url path] copy];
-    // id host = [NSHost hostWithName:hostName];
+        _path.attach([[url path] copy]);
+    }
 
     return self;
 }
 
 - (id)startLoading {
```

Comment:
there is no functional change. 

---
In reply to: [83974374](https://github.com/Microsoft/WinObjC/pull/1175#discussion_r83974374) [](ancestors = 83974374)